src/main/java/guru/qa/allure/notifications/clients/Notification.java

```diff
@@ -1,19 +1,23 @@
 package guru.qa.allure.notifications.clients;
 
+import java.util.List;
+
 import guru.qa.allure.notifications.config.Config;
 import guru.qa.allure.notifications.exceptions.MessagingException;
 import lombok.extern.slf4j.Slf4j;
 @Slf4j
 public class Notification {
 
     public static void send(Config config) throws MessagingException {
-        final Notifier notifier = ClientFactory.from(config);
-        log.info("Sending message...");
-        if (config.getBase().getEnableChart()) {
-            notifier.sendPhoto();
-        } else {
-            notifier.sendText();
+        final List<Notifier> notifiers = ClientFactory.from(config);
+        for (Notifier notifier: notifiers) {
```

Comment:
What will happen if 3 notifiers are configured but the second one is throwing exception?